appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockLineChart.java

```diff
@@ -1,19 +1,10 @@
 package com.google.appinventor.client.editor.simple.components;
 
 import com.google.appinventor.client.editor.simple.SimpleEditor;
-import com.google.appinventor.components.common.ComponentConstants;
-import org.pepstock.charba.client.LineChart;
-import org.pepstock.charba.client.data.DataPoint;
-import org.pepstock.charba.client.data.Labels;
-import org.pepstock.charba.client.data.LineDataset;
-import org.pepstock.charba.client.enums.CubicInterpolationMode;
-import org.pepstock.charba.client.enums.SteppedLine;
-import org.pepstock.charba.client.resources.EmbeddedResources;
-import org.pepstock.charba.client.resources.ResourcesType;
+import com.google.appinventor.client.widgets.dnd.DragSource;
+import org.pepstock.charba.client.ScatterChart;
 
-import java.util.ArrayList;
-
-public final class MockLineChart extends MockChart<LineChart> {
+public final class MockLineChart extends MockChart<ScatterChart> {
```

Comment:
General question: Are GWT tests integrated into the build?

I always prefer to have automated tests for most of the functionality, but otherwise I will just resort to manual testing for GWT.